### PR TITLE
Hide the 'loading' bubbles in chatbot samples

### PR DESCRIPTION
**Pull request by jmartisk**
replaces https://github.com/quarkiverse/quarkus-langchain4j/pull/890

### CODE REVIEW

**Review by geoand — APPROVED** [review]
🎉

File: samples/chatbot-web-search/src/main/resources/META-INF/resources/components/demo-chat.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ export class DemoChat extends LitElement {
                 right: false,
                 sender: {name: 'Bob', id: '007'}
             });
+            chatBot.hideAllLoading();
```
Comment (cescoffier) [inline samples/chatbot-web-search/src/main/resources/META-INF/resources/components/demo-chat.js:15]: I would hide first and display after (just move this before the addMessage)

**Review by cescoffier — CHANGES_REQUESTED** [review]

File: samples/chatbot-web-search/src/main/resources/META-INF/resources/components/demo-chat.js (hunk 1 above)
Comment (jmartisk) [reply samples/chatbot-web-search/src/main/resources/META-INF/resources/components/demo-chat.js]: Ok done

**Review by cescoffier — APPROVED** [review]
Little known fact: Did you know that this version will let us handle streaming (with a bit of javascript, but nothing too hard :-)

**Comment by geoand** [comment]
That's great!

**Pull request merged**